vba/README.md

```diff
@@ -0,0 +1,4 @@
+## This README is for nexar supply VBA example
+
+### Start-up
+* CHANGE FILEPATH TO YOUR FILEPATH OF YOUR API KEY
```

Comment:
Can you add more specific instructions?

Also maybe some screenshots too would be useful.  See https://github.com/NexarDeveloper/nexar-supply-excel-cs as an example.  Doesn't have to be quite that long, but give some background and how to use it.